packages/iris-grid/src/AdvancedFilterCreatorSelectValueList.tsx

```diff
@@ -85,14 +119,16 @@ class AdvancedFilterCreatorSelectValueList extends PureComponent {
     }
   }
 
-  componentWillUnmount() {
+  componentWillUnmount(): void {
     this.stopListening();
   }
 
-  handleSelect(itemIndex, value) {
+  list: SelectValueList<T> | null;
+
+  handleSelect(itemIndex: number, value: T): void {
     const { invertSelection } = this.props;
     let { selectedValues } = this.state;
-    selectedValues = [].concat(selectedValues);
+    selectedValues = ([] as T[]).concat(selectedValues);
```

Comment:
Just do `[...selectedValues]`
```suggestion
    selectedValues = [...selectedValues];
```